test/integration_test/chaosctl/debug.sh

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env bash
```

Comment:
I think this is OK. But I prefer python script . 
Anyway , this doesn't matter.